raresGabriel01/home-assignment
# Issue 6

# Issue link
#6 
# Description
Basic sorting implemented.
Sortable headers are denoted by the `sortable` input on the `t-column` components.
If the `sortable` input on the `t-grid` is `false`, then we can't sort anything within the table, regardless of the `sortable` input on the `t-column`.
The sorting direction of a column is indicated by the arrow next to the header text.
Also, we added an `EventEmitter<SortChangeEvent>` that gets triggered whenever we change the sorting direction.
# How to test?

1. Open the storybook
2. Go to the sortable table story
3. Click on sortable headers (such as name or points) and ensure the table gets sorted in the direction indicated by the arrow
4. Switch between the columns to ensure the direction resets
5. Go to the non sortable table
6. Click on every header and ensure nothing gets sorted

# Attachments
